grails-app/views/inventory/_filters.gsp

```diff
@@ -1,7 +1,7 @@
 <div class="filters" >
 	<g:form method="GET" controller="inventory" action="browse">
 		<g:hiddenField name="max" value="${params.max?:10 }"/>
-		<g:hiddenField name="location.id" value="${session.warehouse.id}"/>
+		<g:hiddenField name="location" value="${session.warehouse.id}"/>
```

Comment:
I don't like this solution because it suggests we don't really know enough to do it properly. I want to make sure we completely understand the problem as well as the ramifications of this type of change. 

In other words, once we start doing it this way we'll need to do it everywhere and that's going to be a lot of work and a lot of re-testing. If the Grails 3 data binding is not working the way it was in Grails 1, then we need to know what has changed and how we should support it.

Because of that, I think we should remove this change and add a new ticket to explore this issue.